userspace/libscap/userspace_flag_helpers.h

```diff
@@ -35,7 +43,9 @@
 #define UDIG
 #endif
 
+#if !defined(_WIN32)
```

Comment:
do we want them for Mac or WASM?

Reply:
It could makes sense to remove it for all the platforms other than linux, wdyt?

Reply:
yeah, I would go for it!